<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadTextureDialog</class>
 <widget class="QDialog" name="LoadTextureDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>190</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>520</width>
    <height>170</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>760</width>
    <height>220</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Load Texture</string>
  </property>
  <layout class="QVBoxLayout" name="rootLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>

   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>28</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>28</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    background-color: #303645;
    color: #a4b0bd;
    font-weight: 600;
    padding-left: 8px;
}</string>
     </property>
     <property name="text">
      <string>Load Texture</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </item>

   <item>
    <layout class="QVBoxLayout" name="bodyLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>

     <item>
      <layout class="QHBoxLayout" name="fileRow">
       <property name="spacing">
        <number>8</number>
       </property>
       <item>
        <widget class="QLabel" name="fileLabel">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>File:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="fileLineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="browseButton">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>

     <item>
      <layout class="QHBoxLayout" name="nameRow">
       <property name="spacing">
        <number>8</number>
       </property>
       <item>
        <widget class="QLabel" name="nameLabel">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nameLineEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="colorSpaceLabel">
         <property name="text">
          <string>Color Space:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="colorSpaceCombo">
         <item>
          <property name="text">
           <string>sRGB</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Linear</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>

     <item>
      <widget class="QLabel" name="hintLabel">
       <property name="styleSheet">
        <string notr="true">QLabel { color: #6b7580; padding-left: 2px; }</string>
       </property>
       <property name="text">
        <string>Tip: sRGB for albedo/emissive, Linear for normal/roughness/metallic/AO.</string>
       </property>
      </widget>
     </item>

     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>

     <item>
      <layout class="QHBoxLayout" name="buttonRow">
       <property name="spacing">
        <number>8</number>
       </property>
       <item>
        <spacer name="buttonSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="loadButton">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Load</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>

    </layout>
   </item>

  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
